portworx-manual/strimzi-px-install.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+# export AZURE_TENANT_ID=""
+# export AZURE_CLIENT_ID=""
+# export AZURE_CLIENT_SECRET=""
+
+echo "Creating namespace kafka"
+kubectl create namespace kafka
+
+
+echo "Installing Portworx"
+
+# Create a secret to give Portworx access to Azure APIs
+kubectl create secret generic -n kube-system px-azure --from-literal=AZURE_TENANT_ID="${AZURE_TENANT_ID}" \
+                                                      --from-literal=AZURE_CLIENT_ID="${AZURE_CLIENT_ID}"\
+                                                      --from-literal=AZURE_CLIENT_SECRET="${AZURE_CLIENT_SECRET}"
+
+
+# Generate custom specs for your portworx config. By default, the script uses Premium volume types, 150 GB, Auto Data and 
+# Management network interfaces with Stork, GUI enabled. To customize this config use the api URL to download a custom yaml
+# [https://docs.portworx.com/portworx-install-with-kubernetes/cloud/azure/aks/2-deploy-px/#]
+
+kubectl apply -f temp-px-deploy/px-gen-spec.yaml
+
+until kubectl get pods --all-namespaces | grep -E "kube-system(\s){3}portworx.*1\/1\s*Running+"
```

Comment:
Perhaps add a comment to say you want to make sure all the Portworx pods in the Daemonset are running